app/controllers/admin/delayed_jobs_controller.rb

```diff
@@ -11,7 +11,7 @@ class DelayedJobsController < ApplicationController
     before_action :set_job, only: [:update, :destroy]
 
     def index
-      @jobs = job_scope.all.order(id: :asc)
+      @jobs = job_scope.all.order(priority: :asc, queue: :asc)
```

Comment:
Based on https://stackoverflow.com/questions/6118540/exact-order-of-job-execution-for-rails-delayed-job it looks like DJ uses `run_at` as an ordering function, maybe include that in the sort?

Reply:
Sure, we almost never (never?) set run_at explicitly, so it probably doesn't matter in this case, but no harm.